### PR TITLE
@alloy => warnings + Podfile cleanup

### PR DESCRIPTION
**Pull request by orta**
So, a few points:

* Looks like I messed up slightly WRT to the ARLoggers, they needed to be `OBJC_` loggers:

``` diff
-#define ARLogInfo(frmt, ...) LOG_MAYBE(NO LOG_LEVEL_DEF, ARLogContextInfo, 0, nil, __PRETTY_FUNCTION__, frmt, ##__VA_ARGS__)
+#define ARLogInfo(frmt, ...) LOG_OBJC_MAYBE(NO LOG_LEVEL_DEF, (DDLogFlag)ARLogContextInfo, 0, nil, __PRETTY_FUNCTION__, frmt, ##__VA_ARGS__)
```
* And we needed to cast the log level.
* Next the migration to frameworks has exposed an annoyance with the way that Mantle/RAC vend their common library `libextobjc`, I don't expect improvements on this front, so for now in-app `@weakify`/`@strongify` has an underscore. I'd also like to see us migrate away from this. Now it's easy to find.

* Removed specific versions from the Podfile. Leaving the `Podfile.lock` to do it's thing. @1aurabrown - be careful with `pod update`.
 * I've added a few more categories in the Podfile, and ordered them alphabetically internally.

### CODE REVIEW

File: Artsy/App/Watch/ARWatchBidNetworkModel.h [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 @interface ARWatchBidNetworkModel : NSObject
 
-+ (void)bidWithDetails:(WatchBiddingDetails *)details:(void (^)(BidderPosition *))success failure:(void (^)(NSError *))failure;
++ (void)bidWithDetails:(WatchBiddingDetails *)details :(void (^)(BidderPosition *))success failure:(void (^)(NSError *))failure;
```
Comment (alloy) [inline Artsy/App/Watch/ARWatchBidNetworkModel.h]: In the words of @ashfurrow: “boo hiss”

Comment (alloy) [inline Artsy/App/Watch/ARWatchBidNetworkModel.h]: Oh wait, what, you’re using anonymous parameters?! You’s crazy, ser.

**Comment by alloy** [comment]
> Next the migration to frameworks has exposed an annoyance with the way that Mantle/RAC vend their common library libextobjc

Can you elaborate on what wasn’t working? It seemed to work, no? Or was it purely a matter of warnings?

**Comment by orta** [comment]
Each library had it's own definition of `@weakify` or `@strongify` - when used as a framework this was being exposed to app code, and we would get "ambiguous definition for weakify" whereas before I think the private/public scoping of the headers worked to keep RACs weakify, and Mantle's weakify from creeping into app code, where we used to explicitly use libextobjc's weakify. 

Now under the hood we use RACs weakify via the _weakify

File: Artsy/App/Watch/ARWatchBidNetworkModel.h (hunk 1 above)
Comment (orta) [inline Artsy/App/Watch/ARWatchBidNetworkModel.h]: looks like I always was, but was doing it ambiguously, and now it's explicit.

**Comment by alloy** [comment]
Gotcha.

**Comment by alloy** [comment]
Those damn 3 tests. There’s a ticket for those, so merging!

**Pull request merged**